### DISCUSSION TITLE
New MVN release?

### DISCUSSION THREAD

**Discussion by benjamij** [JSQLParser/JSqlParser#1227] [General]
Hey all - just wanted to see what the plans were for releasing the latest fixes and improvements? Looks like the latest release on MVN was January...?

**Comment by wumpz** [comment]
Not yet. I think some things for complex expressions still have to be fixed or improved. ( or is this pressing )

**Reply by benjamij** [reply]
Thanks. No, no hurry.

**Reply by manticore-projects** [reply]
> Not yet. I think some things for complex expressions still have to be fixed or improved. ( or is this pressing )

Certainly not pressing.
At the same time, we saw some massive improvements in 4.1 so far and I would actually appreciate an early release. We can also factor in more improvements into 4.2 after.

Release early and often was always a good policy.

**Reply by wumpz** [reply]
Agreed
